File: znld-web/src/main/java/com/sybd/znld/controller/user/dto/LogoutResult.java
package com.sybd.znld.controller.user.dto;

import com.sybd.znld.core.BaseApiResult;
import io.swagger.annotations.ApiModel;


@ApiModel(value = "登出返回数据")
public class LogoutResult extends BaseApiResult{
    public LogoutResult(){}
    public LogoutResult(Integer code, String msg){
        super(code, msg);
    }
    public static LogoutResult fail(String msg){
        return new LogoutResult(1, msg);
    }
    public static LogoutResult success(){
        return new LogoutResult(0, "");
    }
}

File: znld-web/src/main/java/com/sybd/znld/controller/user/dto/LogoutResult.kt
package com.sybd.znld.controller.user.dto;

import com.sybd.znld.core.BaseApiResult;
import io.swagger.annotations.ApiModel;


@ApiModel(value = "登出返回数据")
class LogoutResult(code: Int, msg: String): BaseApiResult(code, msg) {
    companion object {
        @JvmStatic fun fail(msg: String): LogoutResult{
            return LogoutResult(1, msg)
        }
        @JvmStatic fun success(): LogoutResult{
            return LogoutResult(0, "")
        }
    }
}
